internal/schema/stacks/1.9/orchestrate_block.go

```diff
@@ -29,6 +41,22 @@ func orchestrateBlockSchema() *schema.BlockSchema {
 				Description:            lang.PlainText("Rule Name"),
 			},
 		},
+		DependentBody: map[schema.SchemaKey]*schema.BodySchema{
+			schema.NewSchemaKey(schema.DependencyKeys{
+				Labels: []schema.LabelDependent{
+					{Index: 0, Value: "auto_approve"},
+				},
+			}): {
+				// TODO: determine if we need to add any attributes here or Body suffices
```

Comment:
```suggestion
```
Since both types of block have the same body, we don't need to add anything here.